translations/en.json

```diff
@@ -643,6 +646,11 @@
                                         "password": "Password"
                                     }
                                 },
+                                "select_mfa_module": {
```

Comment:
You should modify en.json file under `/src/translation` folder, not this one.

Also, I haven't figure out the way to better handle mfa related string resources, so you have to copy those new strings into each auth_provider.

Reply:
https://github.com/home-assistant/home-assistant-polymer/blob/678016cfc6359c5e94ca6fb1f6e264dafeade0b0/src/translations/en.json#L77